proto/stride/stakeibc/callbacks.proto

```diff
@@ -11,50 +11,37 @@ message SplitDelegation {
 }
 
 message DelegateCallback {
-  string hostZoneId = 1; 
-  uint64 depositRecordId = 2;
-  repeated SplitDelegation splitDelegations = 3;
+  string host_zone_id = 1;
+  uint64 deposit_record_id = 2;
+  repeated SplitDelegation split_delegations = 3;
 }
 
 // ---------------------- Claim Callbacks ---------------------- //
 
 message ClaimCallback {
-  string userRedemptionRecordId = 1;
-  string chainId = 2;
-  uint64 epochNumber = 3;
+  string user_redemption_record_id = 1;
+  string chain_id = 2;
+  uint64 epoch_number = 3;
 }
 
 // ---------------------- Reinvest Callback ---------------------- //
 message ReinvestCallback {
-  cosmos.base.v1beta1.Coin reinvestAmount = 1 [
+  cosmos.base.v1beta1.Coin reinvest_amount = 1 [
     (gogoproto.nullable) = false,
     (gogoproto.castrepeated) = "github.com/cosmos/cosmos-sdk/types.Coin"
   ];
-  string hostZoneId = 3;
+  string host_zone_id = 3;
 }
 
 // ---------------------- Undelegation Callbacks ---------------------- //
 message UndelegateCallback {
-  string hostZoneId = 1; 
-  repeated SplitDelegation splitDelegations = 2;
-  repeated uint64 epochUnbondingRecordIds = 3;
+  string host_zone_id = 1;
+  repeated SplitDelegation split_delegations = 2;
+  repeated uint64 epoch_unbonding_record_ids = 3;
 }
 
 // ---------------------- Redemption Callbacks ---------------------- //
 message RedemptionCallback {
-  string hostZoneId = 1; 
-  repeated uint64 epochUnbondingRecordIds = 2;
-}
-
-// ---------------------- Validator Rebalance Callbacks ---------------------- //
-
-message Rebalancing {
```

Comment:
why is this removed?

Reply:
right, our notional-labs/stride is out of date